src/transition.cpp

```diff
@@ -391,23 +385,53 @@ void Transition::Draw(Bitmap& dst) {
 }
 
 void Transition::Update() {
-	if (IsActive()) {
-		if (flash_iterations > 0) {
+	if (!IsActive()) {
+		return;
+	}
+
+	// FIXME: Break this dependency on DisplayUI
```

Comment:
this FIXME was not supposed to be moved. No DisplayUI here

Reply:
But it is used a few lines down to create the bitmaps.